scripts/setup-modules.sh

```diff
@@ -3,14 +3,10 @@
 set -euo pipefail
 
 SRCDIR=$1
-BUILDDIR=$2
-MODULE_PLAYGROUND=$3
+MODULE_PLAYGROUND=$2
 
-# TODO: meson allows only out of tree builds
-if test "$SRCDIR" != "$BUILDDIR"; then
-    mkdir -p "$MODULE_PLAYGROUND"
-    cp --archive "$SRCDIR/$MODULE_PLAYGROUND/"* "$MODULE_PLAYGROUND/"
-fi
+mkdir -p "$MODULE_PLAYGROUND"
+cp --archive "$SRCDIR/$MODULE_PLAYGROUND/"* "$MODULE_PLAYGROUND/"
```

Comment:
this relies on $MODULE_PLAYGROUND being relative and $CWD being the builddir... Humn.. we could have dropped the $BUILDDIR arg long ago and just check $SRCDIR vs $CWD?